### ISSUE TITLE
Bidirectional wrapper doesn't allow initial state to be specified as keyword argument or in input list

### ISSUE THREAD

**Issue by bchu** [keras-team/keras#8712]
Please make sure that the boxes below are checked before you submit your issue. If your issue is an implementation question, please ask your question on [StackOverflow](http://stackoverflow.com/questions/tagged/keras) or [join the Keras Slack channel](https://keras-slack-autojoin.herokuapp.com/) and ask there instead of filing a GitHub issue.

Thank you!

- [x] Check that you are up-to-date with the master branch of Keras. You can update with:
pip install git+git://github.com/fchollet/keras.git --upgrade --no-deps

- [x] If running on TensorFlow, check that you are up-to-date with the latest version. The installation instructions can be found [here](https://www.tensorflow.org/get_started/os_setup).

- [ ] If running on Theano, check that you are up-to-date with the master branch of Theano. You can update with:
pip install git+git://github.com/Theano/Theano.git --upgrade --no-deps

- [ ] Provide a link to a GitHub Gist of a Python script that can reproduce your issue (or just copy the script here if it is short).

This is a duplicate of: #7365.

`Bidirectional.call(self, inputs, training=None, mask=None):` does not accept an `initial_state` keyword argument.

Furthermore, invoking `Bidirectional.call` with a list of inputs doesn't allow you to specify the initial state either, since `Recurrent.call` only has this code to handle list inputs:

```
if isinstance(inputs, list):
    inputs = inputs[0]
```

I would fix this myself but I'm not sure what the right solution is. My guess is to switch `Bidirectional` to use `self.forward_layer.__call__` instead of `self.forward_layer.call`.
